shared/feature/feedings/src/main/java/com/epmedu/animeal/feedings/presentation/FeedingsScreenEvent.kt

```diff
@@ -0,0 +1,10 @@
+package com.epmedu.animeal.feedings.presentation
+
+import com.epmedu.animeal.feedings.presentation.model.FeedingItem
+
+internal sealed interface FeedingsScreenEvent {
+
+    val feedingItem: FeedingItem
```

Comment:
Don't make this property for the interface, not all events are gonna override it.